client/src/components/client-page/index.jsx

```diff
@@ -0,0 +1,232 @@
+// import * as React from "react";
+// import Table from "@mui/material/Table";
+// import TableBody from "@mui/material/TableBody";
+// import TableCell from "@mui/material/TableCell";
+// import TableContainer from "@mui/material/TableContainer";
+// import TableHead from "@mui/material/TableHead";
+// import TableRow from "@mui/material/TableRow";
+import EditIcon from "@mui/icons-material/Edit";
+import ContentCopyIcon from "@mui/icons-material/ContentCopy";
+import Box from "@mui/material/Box";
+import Paper from "@mui/material/Paper";
+import Chip from "@mui/material/Chip";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+import Typography from "@mui/material/Typography";
+import { Avatar } from "@mui/material";
+import Grid from "@mui/material/Grid";
+import Divider from "@mui/material/Divider";
+
+export default function ClientPage() {
+  const userInfo = {
```

Comment:
The client object has a few other relevant fields here. Specifically `closure_date`, `status_at_exit`, `status_at_3_months`, `status_at_6_months`, `status_at_9_months`, `status_at_12_months`. The attached image shows how some of these fields may be used to display information about the client. Further the status at 3 months, status at 6 months, status at 9 months, status at 12 months rows should only be displayed and editable 3, 6, 9 and 12 months after being closed respectively.
<img width="429" alt="Screenshot 2024-01-09 at 7 07 32 PM" src="https://github.com/uoftblueprint/tces/assets/58835213/18d03fb6-ad03-4763-b3b2-a4a8e4b36512">.
After x months have passed the user should be able to edit the status at x months/status at exit row in the edit-client-page and change it to one of "undefined", "training", "employed" or "no results" in compliance with how those fields are stored in the database: https://github.com/uoftblueprint/tces/wiki/Database-Schema